### ISSUE TITLE
Dynamic import of third party cjs modules does not resolve default exports

### ISSUE THREAD

**Issue by surgicaI** [rollup/rollup#2949]
- Rollup Version: 1.15.6
- Operating System (or Browser): Mac Os
- Node Version: 10.11.0

### How Do We Reproduce?
```js
// library.js
module.exports = function foo() {}
```

```js
// usage.js
let dep = await import('library');
console.log(dep.foo); // undefined
console.log(dep.default.foo); // The `foo` function.
```

### Expected Behavior

`dep.foo` should have the default exported function. As it is congruent to 
`import dep from 'library';`. 

I use `rollup-plugin-commonjs`. It might be a issue with the plugin.

**Comment by ashubham** [comment]
+1 for this issue. 
```js
let dep = await import('library');
```
behaves like `import * as dep from 'library'`. I am not sure of what the spec says here though.

**Comment by lukastaegert** [comment]
> behaves like import * as dep from 'library'. I am not sure of what the spec says here though

That is exactly what the spec says: A dynamic import returns the export namespace of a module, the default export should be accessible via `.default`. I must admit the latter is not the case as for CJS, we should run an interop helper to find out what the default export is (if any?) and put that on `default` (similar to how we run an interop help if you import the default from some module). I.e. this:
https://rollupjs.org/repl?version=1.15.6&shareable=JTdCJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmltcG9ydCUyMG15RGVmYXVsdCUyMGZyb20lMjAnZXh0ZXJuYWwnJTNCJTVDbmNvbnNvbGUubG9nKG15RGVmYXVsdCklM0IlMjIlMkMlMjJpc0VudHJ5JTIyJTNBdHJ1ZSU3RCU1RCUyQyUyMm9wdGlvbnMlMjIlM0ElN0IlMjJmb3JtYXQlMjIlM0ElMjJjanMlMjIlMkMlMjJuYW1lJTIyJTNBJTIybXlCdW5kbGUlMjIlMkMlMjJhbWQlMjIlM0ElN0IlMjJpZCUyMiUzQSUyMiUyMiU3RCUyQyUyMmdsb2JhbHMlMjIlM0ElN0IlN0QlN0QlMkMlMjJleGFtcGxlJTIyJTNBbnVsbCU3RA==

Should log the same as this:
https://rollupjs.org/repl?version=1.15.6&shareable=JTdCJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmNvbnNvbGUubG9nKGltcG9ydCgnZXh0ZXJuYWwnKS5kZWZhdWx0KSUzQiUyMiUyQyUyMmlzRW50cnklMjIlM0F0cnVlJTdEJTVEJTJDJTIyb3B0aW9ucyUyMiUzQSU3QiUyMmZvcm1hdCUyMiUzQSUyMmNqcyUyMiUyQyUyMm5hbWUlMjIlM0ElMjJteUJ1bmRsZSUyMiUyQyUyMmFtZCUyMiUzQSU3QiUyMmlkJTIyJTNBJTIyJTIyJTdEJTJDJTIyZ2xvYmFscyUyMiUzQSU3QiU3RCU3RCUyQyUyMmV4YW1wbGUlMjIlM0FudWxsJTdE

And obviously it does not.

@surgicaI Not sure I understand your example. The library in your example only has a default export which is a function (that happens to have the id foo but this should have no effect on  the exports). So what *should* happen (even though it does not) is that

```js
let dep = await import('library');
console.log(dep); // {default: function foo() {}}
console.log(dep.default); // The `foo` function.
```

Is that what you mean?

**Comment by surgicaI** [comment]
@lukastaegert 
```let dep = await import('library');
console.log(dep); // {default: function foo() {}}
console.log(dep.default); // The `foo` function.```
Yes this is what I meant. Thanks

**Comment by lukastaegert** [comment]
Fix at #2954.
